gitlab/datadog_checks/gitlab/gitlab.py

```diff
@@ -66,6 +61,27 @@ def check(self, instance):
         for check_type in self.ALLOWED_SERVICE_CHECKS:
             self._check_health_endpoint(instance, check_type, custom_tags)
 
+    def _create_gitlab_prometheus_instance(self, instance, init_config):
+        """
+        Set up the gitlab instance so it can be used in OpenMetricsBaseCheck
+        """
+        # Mapping from Prometheus metrics names to Datadog ones
+        # For now it's a 1:1 mapping
+        # TODO: mark some metrics as rate
+        allowed_metrics = init_config.get('allowed_metrics')
```

Comment:
We should probably move this away from `init_config` WDYT? (same for `gitlab_runner`)

Reply:
Let's do this in some follow-up work